usr/src/uts/armv8/io/cbe.c

```diff
@@ -341,6 +347,8 @@ get_cbe_vector(void)
 					uint32_t *interrupts = __builtin_alloca(len);
 					prom_getprop(timer, "interrupts", (caddr_t)interrupts);
 					int index = (num > 1)? 1: 0;
+					if (index == 1 && cpu->cpu_m.mcpu_boot_el == 1)
```

Comment:
Can you leave a comment saying that this is choosing the physical or virtual counter based on boot EL? Just so we don't forget.

Reply:
Done.